DCMParser.java

```diff
@@ -0,0 +1,134 @@
+package dcmparser;
+
+import java.io.File;
+import org.dcm4che3.data.Attributes;
+import org.dcm4che3.data.Tag;
+import org.dcm4che3.io.DicomInputStream;
+import org.postgresql.util.PSQLException;
+
+import daoclasses.ImageDAO;
+import daoclasses.PatientStudyDAO;
+import daoclasses.SeriesDAO;
+import ui.Home;
+
+
+
+public class DCMParser {

```

Comment:
This file, I will review after once we refactor and move to Model View Controller design pattern